GameData/RP-0/Contracts/Milestones/First Science Sat.cfg

```diff
@@ -27,12 +27,8 @@ CONTRACT_TYPE
 		
 	// ************ REWARDS ************
 	prestige = Exceptional   // 1.5x
-	advanceFunds = 117779 * @RP0:globalHardContractMultiplier
-	rewardScience = 0
-	rewardReputation = 40
-	rewardFunds = 55556 * @RP0:globalHardContractMultiplier
-	failureReputation = 40
-	failureFunds = @advanceFunds * 0.5
+	genericReward = 416000
+	genericAdvanceMultiplier = 1.0 / 3.0
```

Comment:
Does that mean that advance is 1/3 of the completion reward? If yes, then the advance used to be twice as big as the reward. Why are you changing that?

Reply:
Ah, my mistake.